hub/src/LayerZeroApp.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity 0.8.12;
+pragma solidity >=0.8.0;
```

Comment:
The compiler declaration is very broad. Let's pick one version and stick with it.